src/components/dialogs/parameters/sensi/sensitivity-analysis-parameters.js

```diff
@@ -544,7 +557,48 @@ export const SensitivityAnalysisParameters = ({
         resetSensitivityParametersAndProvider,
     ]);
 
-    const isMaxReached = () => Math.abs(analysisComputeComplexity) > numberMax;
+    const renderComputingEventLoading = () => {
+        return (
+            <Box sx={styles.modificationsTitle}>
+                <Box sx={styles.icon}>
+                    <CircularProgress
+                        size={'1em'}
+                        sx={styles.circularProgress}
+                    />
+                </Box>
+                <Typography noWrap>
+                    <FormattedMessage id={'computing in progress'} />
```

Comment:
You need to move this to translation files instead of hardcoding it.

Reply:
ok